modular_splurt/code/datums/traits/neutral.dm

```diff
@@ -820,3 +820,55 @@
 	var/obj/item/clothing/mask/gas/cosmetic/gasmask = new(get_turf(quirk_holder)) // Uses a custom gas mask
 	H.equip_to_slot(gasmask, ITEM_SLOT_MASK)
 	H.regenerate_icons()
+
+/datum/quirk/prem
```

Comment:
you'll wanna change the class name to something more descriptive like `hair_trigger`